src/test/java/org/apache/accumulo/access/AuthorizationTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.access;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+public class AuthorizationTest {
+
+  @Test
+  public void testDuplicateExpressions() {
+    // Test that we can de-duplicate a list of String expressions
+    Set<String> expected = Set.of("A&B", "A&C");
+    var authorization = Authorizations.of("A&B", "A&C", "A&B", "A&B", "A&C");
+    assertEquals(expected, authorization.asSet());
```

Comment:
Would be nice to check the other method has the same behavior for dupes. Think the following should do it.

```suggestion
    assertEquals(expected, authorization.asSet());
    authorization = Authorizations.of(List.of("A&B", "A&C", "A&B", "A&B", "A&C"));
    assertEquals(expected, authorization.asSet());
```